examples/BLDC_HALL/README.md

```diff
@@ -0,0 +1,9 @@
+# Setup for Brushlessmotor with Hallsensor
+
+Status: not tested, not working
+Pictures follow.
+
+## Hardware
+Connect the three wires of your brushless motor to the 'U V W' outputs of the board (the order deosn't matter). 
```

Comment:
Hmmmm ... I would say it matters, depending how the code is written. Because how you connect the wires is determining the rotation direction.